ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -3517,7 +3517,7 @@ router_ospf:
           default_information_originate:
             metric: < Integer 1-65535 > # Value of the route metric
             metric_type: < 1 | 2 > # OSPF metric type
-      maximum_paths: < Integer 1-32 >
+      maximum_paths: < Integer 1-64 >
```

Comment:
Any reason to choose 64 rather than the default on most platforms 128? Maybe let's move it directly to 128?

https://www.arista.com/en/um-eos/eos-open-shortest-path-first-version-2#xx1153509

Reply:
yes let's update to 128, vEOS showed 64 so this is why that value was chosen!